### ISSUE TITLE
Feature Request: Support session token

### ISSUE THREAD

**Issue by hitsumabushi** [Reggino/SdbNavigator#27]
# What I want

Please support authentication using Session Token.

# Differences from existing features

Describes the differences from the feature added at https://github.com/Reggino/SdbNavigator/issues/18.

#18  is a function to assume role to the specified stsARN using the user's {access key id, secret key}, and to operate SimpleDB using the new pair of {access key id, secret key, session token} obtained from the user. 
This has been implemented.
In this case, the user has the original credentials for the switch role.

What we would like to request here is the ability for users to have a pair of {access key id, secret key, session token} to operate SimpleDB with their credentials.
In other words, it is like directly specifying the authentication information of the 2nd stage of the feature implemented in #18.
With this feature, it is possible to support the case of wanting to operate as an assume role like in #18.  This is a more commonly applicable feature.

 # Why is it needed?

In organizations where SSO is implemented by an external IdP for AWS login itself, there are cases where a role is assigned when logging in.
In this case, when a user obtains an API key, it can only be obtained with the pair of {access key id, secret key, session token}.

Even in these cases, I'd like to use this useful tool.
